src/components/LogOut/index.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import { Button } from 'react-bootstrap'
+const LogOut = () => {
+  return (
+    <>
+      <Button variant="danger">LogOut</Button>
+    </>
+  )
+}
+export default LogOut
```

Comment:
We already have a logout in the navbar dropdown.

Reply:
I had wrote it for the demo in thesis report, forgot to delete :-)

Reply:
Alright, good idea :)